### PR TITLE
cicd: push tagged image on release

### PR DESCRIPTION
**Pull request by RobPasMue**
As title says. This will create tagged images with the tag name on release

### CODE REVIEW

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -144,8 +144,17 @@ jobs:
...
+          else
+            echo "IS_A_RELEASE=false" >> $GITHUB_ENV
+          fi
+
       - name: Build image and export to GHCR
         uses: docker/build-push-action@v5
+        if: env.IS_A_RELEASE == 'false'
         with:
           context: .
           file: ./docker/Dockerfile
```
Comment (greschd) [inline .github/workflows/ci_cd.yml:160]: The line right below this (which I can't comment on..) is ``push: ${{ github.ref == 'refs/heads/main' }}``. Is that still correct, or does it need a check for tags?

Comment (greschd) [reply .github/workflows/ci_cd.yml:160]: Aha, and the release tag is only pushed below, correct?

File: .github/workflows/ci_cd.yml [hunk 2]
```diff
@@ -156,6 +165,21 @@ jobs:
...
+      - name: Build image and export to GHCR
+        if: env.IS_A_RELEASE == 'true'
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ./docker/Dockerfile
+          push: true
+          target: app_minimal
+          tags: |
+            ${{ env.REGISTRY }}/${{ env.IMAGE_OWNER }}/${{ env.IMAGE_NAME_BASE }}:latest ,
```
Comment (greschd) [inline .github/workflows/ci_cd.yml:177]: Does this push both `latest` and `x.y.z`, or only `x.y.z`?

File: .github/workflows/ci_cd.yml (hunk 1 above)
Comment (RobPasMue) [reply .github/workflows/ci_cd.yml:160]: Yep that's right

File: .github/workflows/ci_cd.yml (hunk 2 above)
Comment (RobPasMue) [reply .github/workflows/ci_cd.yml:177]: Both of them

Comment (RobPasMue) [reply .github/workflows/ci_cd.yml:177]: Would you only prefer ``x.y.z``?

Comment (greschd) [reply .github/workflows/ci_cd.yml:177]: I think so, yeah. That way we can do path releases on old release branches without messing up `latest`.

Comment (RobPasMue) [reply .github/workflows/ci_cd.yml:177]: Okay sure! On it =)

**Review by greschd — APPROVED** [review]

**Pull request merged**